xblock/test/test_core.py

```diff
@@ -3,16 +3,18 @@
 Tests the fundamentals of XBlocks including - but not limited to -
 metaclassing, field access, caching, serialization, and bulk saves.
 """
+from __future__ import unicode_literals
+from builtins import next, str  # pylint: disable=redefined-builtin
```

Comment:
Do you know why reimporting `next` is required?  That one should just work in python 2, right?